.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -5,38 +5,109 @@ body:
 
-        Before filing a bug, please be sure you have searched through
-        [existing bugs](https://github.com/open-telemetry/opentelemetry-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug)
-        to see if your bug is already addressed.
-
-        If your bug is related to an instrumentation or plugin in [opentelemetry-js-contrib](https://github.com/open-telemetry/opentelemetry-js-contrib)
-        please be sure to file it there.
+        Before filing a bug, please be sure you have searched through [existing open bugs](https://github.com/open-telemetry/opentelemetry-js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug)
+        to see if this bug has already been filed.
+  - type: markdown
+    attributes:
+      value: |
+        ## Basic Troubleshooting
+  - type: checkboxes
+    id: updated-to-latest
+    attributes:
+      description: |
+        > [!NOTE]
+        > If you depend on experimental packages with a range like `opentelemetry/sdk-node@^0.53.0` that is equivalent
+        > to `@opentelemetry/sdk-node@0.53.x`, NOT `@opentelemetry/sdk-node@0.x.x`. `0.54.x` **WILL NOT** be picked up
+        > by this constraint due to the way semantic versioning works for `0.x` packages. This is intentional, see
+        > https://semver.org/ for details.
+      label: Updating packages
+      options:
+        - label: I have updated all `@opentelemetry/*` packages but the issue persists
+          required: true
+  - type: dropdown
+    id: esm
+    attributes:
+      description: |
+        **How to find this information:**
+          - check your `package.json`
+            - if you have `"type": "commonjs"` or no `"type"` then you're likely using CommonJS
+            - if you have `"type": "module"`, you are likely using ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.
+          - other indicators:
+            - using `.js` or `.cjs` (CommonJS)
+            - using `.mjs` (ESM - please ensure you have read the [instructions for ESM users](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) before opening an issue.)
+      label: Which module type are you using?
+      multiple: false
+      options:
+        - CommonJS
+        - ESM
+        - I don't know
+  - type: checkboxes
+    id: bundling
+    attributes:
+      description: |
+        Bundling removes `require` and `import` statements, which results in most `@opentelemetry/instrumentation-*` packages not working as they rely on the presence
+        of these statements to function.
+      label: Bundling and `@opentelemetry/instrumentation-*`packages
+      options:
+        - label: I understand that `@opentelemetry/instrumentation-*` packages are not intended to work in bundles apps (with a few exceptions).
+          required: true
```

Comment:
I wonder if `required: true` is a little harsh here for a report on something that has nothing to do with bundling.

Reply:
We can also leave it out for now, WDYT?

Unfortunately, there's no mechanism that allows us to conditionally show a new checkbox when another was checked. I'd be pretty cool to have a checkbox "I'm using a bundler" which makes another checkbox pop up that says "I understand that..." and a dropdown that asks which bundler they are using (for any problems that are not instrumentation-package related).

Reply:
Agreed that we can remove the checkbox for now so it doesn't impact non-bundler users reporting a bug

Reply:
I removed it now, moving that to a troubleshooting guide that I'll link form here in a follow-up PR.